forecast-admin/forecast/opportunities/models.py

```diff
@@ -12,7 +13,7 @@ def __str__(self):
         return "%s (%s)" % (self.organization, self.region)
 
 
-class Award(models.Model):
+class Opportunity(models.Model):
     AWARD_STATUS_CHOICES = (
         ("Awarded", "Awarded"), ("Award Pending", "Award Pending"),
```

Comment:
In a CHOICES like this, the 0 index of the tuple can be shorter because that's what's stored in the database. No need to store the whole string in the DB. So, you could do ('A', 'Awarded'), ('AP', 'Award Pending') or whatever makes sense. Also, this will make the award status field shorter. You could also assign them integers, since it's a sequence.